packages/client/src/components/ratings/FutureProofRatingView/FutureProofRatingView.tsx

```diff
@@ -19,16 +29,42 @@ type ComponentProps = Pick<
  * @returns {View} The FutureProof Rating (breakdown) view
  */
 
-const FutureProofRatingView = ({ sustainabilityScore, sustainabilityCertificates, sustainabilityRatings } : ComponentProps ) => {
+const FutureProofRatingView = ({ route } : Props ) => {
+
+  const ratings = []
+  const ratingsNames = []
+
+  ratings.push(route.params.businessToDisplay.humanRightsScore)
+  ratings.push(route.params.businessToDisplay.carbonScore)
+  ratings.push(route.params.businessToDisplay.certificateScore)
+  ratings.push(route.params.businessToDisplay.envProtectionScore)
+  ratings.push(route.params.businessToDisplay.diversityScore)
+  ratings.push(route.params.businessToDisplay.productSafetyScore)
+  ratings.push(route.params.businessToDisplay.equalPayScore)
+  ratings.push(route.params.businessToDisplay.taxScore)
+  ratings.push(route.params.businessToDisplay.dataPrivacyScore)
+  ratings.push(route.params.businessToDisplay.charitableScore)
+  
+  ratingsNames.push('Human Rights Score')
+  ratingsNames.push('Carbon Score')
+  ratingsNames.push('Certificate Score')
+  ratingsNames.push('ENV Protection Score')
+  ratingsNames.push('Diversity Score')
+  ratingsNames.push('Product Safety Score')
+  ratingsNames.push('Equal Pay Score')
+  ratingsNames.push('Tax Score')
+  ratingsNames.push('Data Privacy Score')
+  ratingsNames.push('Charitable Score')
```

Comment:
Could you perhaps use a map over the object entries? There must be a more readable way of expressing this